app/src/views/GrantRegistryGrantDetail.vue

```diff
@@ -72,6 +72,39 @@
             errorMsg="Please enter a description"
           />
 
+          <!-- Grant website -->
+          <BaseInput
+            v-model="form.website"
+            description="Your grant's website"
+            id="grant-website"
+            label="Grant website"
+            :rules="isValidUrl"
+            errorMsg="Please enter a valid URL"
+            :required="false"
+          />
+
+          <!-- Grant github -->
+          <BaseInput
+            v-model="form.github"
+            description="Your grant's github"
+            id="grant-github"
+            label="Grant github"
+            :rules="isValidUrl"
```

Comment:
In addition to validating that this is a URL, I think we should validate specifically that it's a *GitHub* URL. Counter argument here would be that some projects might use GitLab or something else.

Reply:
Hmm, I think we should leave this as validating a generic URL. Because there's also Bitbucket, Radicle, and probably others. And Radicle is a decentralized git so there may not be a single 2LD we can look for to cover all Radicle repos